src/components/DirectoryItemSelector/directory-item-selector.tsx

```diff
@@ -256,6 +265,8 @@ const DirectoryItemSelector: FunctionComponent<DirectoryItemSelectorProps> = ({
                 validationButtonText={validationButtonText}
                 className={className}
                 cancelButtonProps={cancelButtonProps}
+                selected={selected}
```

Comment:
if not used/managed in this `DirectoryItemSelector` then maybe we could improve this code by forwarding props to `TreeViewFinder`
```
<TreeViewFinder {...treeViewFinderProps}/>
```

Reply:
done